src/views/Entries/FilterEntriesForm/index.js

```diff
@@ -32,26 +32,28 @@ import {
     projectDetailsSelector,
     entryFilterOptionsForProjectSelector,
     setEntryFilterOptionsAction,
+    organizationTypesSelector,
+    setOrganizationTypesAction,
 } from '#redux';
 import _ts from '#ts';
 
 import FrameworkFilter from './FrameworkFilter';
 
 import styles from './styles.scss';
 
-const emptyList = [];
-
 const mapStateToProps = state => ({
     activeProject: activeProjectIdFromStateSelector(state),
     entriesFilters: entriesViewFilterSelector(state),
     entryFilterOptions: entryFilterOptionsForProjectSelector(state),
+    organizationTypes: organizationTypesSelector(state),
     projectDetails: projectDetailsSelector(state),
 });
 
 const mapDispatchToProps = dispatch => ({
     setEntriesViewFilter: params => dispatch(setEntriesViewFilterAction(params)),
     updateEntriesViewFilter: params => dispatch(updateEntriesViewFilterAction(params)),
     setEntryFilterOptions: params => dispatch(setEntryFilterOptionsAction(params)),
+    setOrganizationTypes: params => dispatch(setOrganizationTypesAction(params)),
```

Comment:
Lets not keep this in redux.
As we are fetching this everytime and only used here.